### DISCUSSION TITLE
How to derive censoring with MIMIC III?

### DISCUSSION THREAD

**Discussion by diego-s** [MIT-LCP/mimic-code#1269] [MIMIC-III]
Hi all,
I have a question on how to derive appropriate time to event and censoring columns for a survival analysis with MIMIC III. Would the death time and discharge time from the ADMISSIONS table be the right columns to use for this? E.g. if there is a death time, time = death time and status = 1, if not time = discharge time and status = 0? I assume it may be helpful to derive a "time to last follow up" variable by also looking at other dates in other tables that may indicate that the patient was still alive at that time? Any insights on this would be helpful.
Thanks.
Best regards,
Diego

**Comment by tompollard** [comment]
@diego-s adding a censored date (i.e. date after which data is no longer available) is an interesting idea. We'll add this to our list of things to look into. For MIMIC-III v1.4, see: https://github.com/MIT-LCP/mimic-code/issues/647

**Comment by diego-s** [comment]
Thanks Tom, is my interpretation correct here that the current best practice would be to censor the patients ~82 days (more details are shown in the thread you shared) after hospital discharge?
